packages/deployments/contracts/contracts/nomad-xapps/contracts/XAppConnectionClient.sol

```diff
@@ -25,7 +25,7 @@ abstract contract XAppConnectionClient is OwnableUpgradeable {
 
   // ======== Initializer =========
 
-  function __XAppConnectionClient_initialize(address _xAppConnectionManager) internal initializer {
+  function __XAppConnectionClient_initialize(address _xAppConnectionManager) internal onlyInitializing {
```

Comment:
we should avoid changing these functions as eventually we will want to import the nomad contracts from the nomad packages

Reply:
(tried recently and was still having some issues)

Reply:
yeah, but that is old version of openzeppelin.  they might have to update that later. 
anyway i will revert that for now.